docs/cheaha/software/modules.md

```diff
@@ -52,6 +52,42 @@ If you want to revert to the default modules, you can use:
 module reset
 ```
 
+## Saving Modules using Collections
+
+To save time in typing in long list of modules everytime you work on a project, you can save the desired list of modules using module collection. To acheive this first load the desired modules, and save it using a collection name as below,
+
+```bash
+module load module 1 module 2 ...
```

Comment:
`module_1` and `module_2` with underscores will help with clarity.

Reply:
Added underscores.